src/components/SliderHomePage/SliderHomePage.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable max-len */
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { useState } from 'react';
+import { useSwipeable } from 'react-swipeable';
```

Comment:
code compiles with error 

> Can't resolve 'react-swipeable'

could you install `'react-swipeable'` and push updated package.json